server/app/services/migration/ProgramMigrationService.java

```diff
@@ -84,18 +95,21 @@ public ErrorAnd<ProgramMigrationWrapper, String> deserialize(String programJson)
 
   /**
    * Checks if there are existing questions that match the admin id of any of the incoming
-   * questions. If a match is found, generate a new admin name of the format "orginal admin name-n".
+   * questions. If a match is found, generate a new admin name of the format "orginal admin name -
+   * a".
```

Comment:
**nitpick:** Can you put these on the same line? maybe move down the whole admin name